### PR TITLE
feat: Added floordiv method in tensorflow frontend and unit test for it.

### LINKED ISSUE #26463: floordiv

### PR DESCRIPTION
**Pull request by Dheeraj-Bal**
# PR Description

 Added floordiv method in TensorFlow frontend in tensor.py file and unit test for it.

## Related Issue

Close #26463

## Checklist

- [ ] Added a function

### CODE REVIEW

**Comment by Dheeraj-Bal** [comment]
Hi @hello-fri-end , Can you please guide me on how to pass all the checks and what am I doing wrong with my changes? It would be really helpful.

File: ivy/functional/frontends/tensorflow/tensor.py [hunk 1]
```diff
@@ -53,6 +53,9 @@ def shape(self):
     def get_shape(self):
         return tf_frontend.raw_ops.Shape(input=self)
 
+    def floordiv(self, x, y, name="floorviv"):
+        return tf_frontend.raw_ops.FloorDiv(x, y, name)
+
```
Comment (hello-fri-end) [inline ivy/functional/frontends/tensorflow/tensor.py:58]: Why are we using the `raw_ops` `FloorDiv` function here and not the frontend `tf.floordiv` which is defined here (https://github.com/unifyai/ivy/blob/0f33aebcbcd443174f27bb8b43f322853e4078a3/ivy/functional/frontends/tensorflow/math.py#L277)

File: ivy_tests/test_ivy/test_frontends/test_tensorflow/test_tensor.py [hunk 2]
```diff
@@ -1465,6 +1465,88 @@ def test_tensorflow__xor__(
...
+    frontend_method_data,
+    init_flags,
+    method_flags,
+    backend_fw,
+    on_device,
+):
+    input_dtype, x = dtype_and_x
+    helpers.test_frontend_method(
+        init_input_dtypes=input_dtype,
+        backend_to_test=backend_fw,
```
Comment (hello-fri-end) [inline ivy_tests/test_ivy/test_frontends/test_tensorflow/test_tensor.py:1493]: Why are we adding two test functions here? Looks like the only difference in these is that in (1) you're calling `helpers.get_dtypes("numeric")` while in the second you're calling `helpers.get_dtypes("float")`. You should actually use `helpers.get_dtypes("valid") and write only one test.

**Review by hello-fri-end — CHANGES_REQUESTED** [review]
Hey @Dheeraj-Bal , thanks for working on this issue. Just left some minor comments, happy to review once again after you're done making the changes. Happy coding :)

**Pull request closed without merging**